packages/gasket-utils/lib/package-manager.js

```diff
@@ -139,7 +139,9 @@ class PackageManager {
    * @public
    */
   async install(args = []) {
-    return this.exec('install', args);
+    // Installing with --legacy-peer-deps flag to accommodate npm7, specifically
+    // requiring different versions of react
+    return this.exec('install', [...args, '--legacy-peer-deps']);
```

Comment:
is this backwards comapatible? aka if i did this w/ `npm@6` will it still add `--legacy-peer-deps`?

Reply:
Should we also add tests around this?

Reply:
`--force` has other implications, whereas `--legacy-peer-deps` handles the specific issue at hand.

Reply:
> is this backwards comapatible? aka if i did this w/ `npm@6` will it still add `--legacy-peer-deps`?. iirc `npm@6` does not have this method. If you want to go this route you will want to check the `npm` version in use prior to this return

I did test this with npm@6 and had no issues installing.